adding_species_manuscript.tex

```diff
@@ -681,13 +683,13 @@ \subsection*{Filling out the missing pieces}
 %Ilan: tried to give a few concrete examples here
 If, for example, the main purpose of the simulation is to examine the distribution of lengths of shared haplotypes between individuals,
 or study patterns of background selection,
-then it makes sense to simulate such pseudo chromosomes.
-Note, however, that specific genetic correlations between different contigs lumped together can be highly inaccurate.
-So, if the main purpose of the simulation is to examine local patterns of genetic variation in loci of interest, then it would be more appropriate to simulate the relevant contigs separately (even if they are short), or to randomly sample several mappings of contigs to pseudo-chromosomes.
-Finally, we note that for some purposes it may be sufficient to simulate a large number of unlinked sites \citep{Gutenkunst2009,Excoffier2013},
```

Comment:
Perhaps this is true, but tangential - I think to say this we should make clear what "some purposes" are, which is beyond the scope of this bit.

Reply:
Good call, but I don't think we're saying people *can't* do single loci? We're not trying to review all possible simulation methods? (Or, if we do, we need some more context?)

Reply:
Let's keep this commented out for now, and see if we miss this in future editing rounds. If needed, we can add this with some more detail, like saying that tasks such as demographic inference and DFE inference can be done using unlinked sites.

Reply:
I think the sentence is both useful and tangential - this paper isn't just about adding species to stdpopsim, it's also promoting standardized, species-specific simulations in general, which will appeal to anyone doing empirical research, including people with eg. RADSeq data. But those people aren't who the paper is for, and I think it's valid to make that pretty explicit. Perhaps "However, this approach would not have the many benefits of whole-chromosome simulations, which we promote in \stdpposim. While some of the same considerations hold, a discussion of such simulations goes beyond the scope of this paper."

Reply:
I think that @petrelharp is raising another issue - that we should not suggest that simulation of independent sites is useful without explicitly saying when it is possibly useful. It's not like people will stop doing independent site simulations, and there are probably a few things that such simulations can benefit from by using our guidelines (QC, fitting a demographic model, mutation rate), but we don't necessarily need to go out of our way to make this point explicitly. My suggestion is to leave this part of the text commented out and return to it in a clean read. If it still looks to us like something's missing here, then we can figure a way to make this point the "right" way.